src/views/Navbar.tsx

```diff
@@ -0,0 +1,31 @@
+import { BsPersonCircle } from 'react-icons/bs';
+import { useLocation } from 'react-router-dom';
+import { CustomLink } from '../components/NavbarLink';
+import '../styles/navbar/navbar.css'
+
+const onLogoutClick = () => {
+  console.log('Kirjaudu ulos clicked.');
+};
+
+export const Navbar = () => {
+  // Don't show navbar on register and login view.
+  const path = useLocation().pathname;
+  return path !== '/' && path !== '/register' ? (
```

Comment:
Tämmösen listan ylläpito voi käydä ärsyttäväksi jos näitä tulee aina lisää. Nopeana vaihtoehtona tuli mieleen tarkastella käyttäjän kirjautumista, mutta se ei ole vielä valmista.

Olkoon näin, mutta lisäisin TODO:n missä ilmaistaisiin tarve refaktoroinnille